### PR TITLE
Do not use dots in string resource names

### PR DESCRIPTION
**Pull request by iverc**

### CODE REVIEW

**Comment by iverc** [comment]
PTAL

File: README.md [hunk 1]
```diff
@@ -388,21 +388,21 @@ Name your strings with keys that resemble namespaces, and don't be afraid of rep
...
+<string name="error_message_call">Call failed</string>
+<string name="error_message_map">Map loading failed</string>
 ```
 
 Don't write string values in all uppercase. Stick to normal text conventions (e.g., capitalize first character). If you need to display the string in all caps, then do that using for instance the attribute [`textAllCaps`](http://developer.android.com/reference/android/widget/TextView.html#attr_android:textAllCaps) on a TextView.
 
 **Bad**
 ```xml
-<string name="error.message.call">CALL FAILED</string>
+<string name="error_message_call">CALL FAILED</string>
```
Comment (alikyo) [inline README.md]: should keep dots here to show the bad practice ;)

**Comment by piotrek1543** [comment]
Maybe I'm missing something, but it's already fixed. Update your PR or close it.

**Comment by iverc** [comment]
Apologies, probably it was fixed after I forked the repo. Closing

**Pull request closed without merging**